core/components/header/index.tsx

```diff
@@ -52,6 +67,32 @@ const getLogo = async () => {
   return data.settings ? logoTransformer(data.settings) : '';
 };
 
+const getCartCount = async () => {
+  const cookieStore = await cookies();
+  const cartId = cookieStore.get('cartId')?.value;
+
+  const customerAccessToken = await getSessionCustomerAccessToken();
+
+  const response = await client.fetch({
+    document: GetCartCountQuery,
+    variables: { cartId },
+    customerAccessToken,
+    fetchOptions: {
+      cache: 'no-store',
+      next: {
+        tags: [TAGS.cart],
+      },
+    },
+    // channelId,
```

Comment:
Do we need to address this later or can we delete?

Reply:
Addressing this.

Reply:
Actually, I don't think this is needed.